documentation/attributes.html

```diff
@@ -41,47 +26,471 @@
   <li><a href="document-summaries.html">document summaries</a></li>
   <li><a href="parent-child.html">document reference</a></li>
 </ul>
-Or, the other way around, use <a href="proton.html#index">index</a> for fields used for text search,
-with <a href="linguistics.html#stemming">stemming</a> and
-<a href="linguistics.html#normalization">normalization</a>.
+<p>
+  Or, the other way around, use <a href="proton.html#index">index</a> for fields used for text search,
+  with <a href="linguistics.html#stemming">stemming</a> and
+  <a href="linguistics.html#normalization">normalization</a>.
 </p><p>
-<em>attribute</em> is a keyword in <a href="reference/schema-reference.html#attribute">
-schemas</a>, specifying the indexing for a document field -
-see the <a href="reference/advanced-indexing-language.html">indexing language</a>.
+  Attributes speed up query execution and document updates, trading off memory.
+  As data structures are regularly optimized, consider both static and temporary resource usage -
+  see <a href="#attribute-memory-usage.html">attribute memory usage</a> below.
+  Use attributes in document summaries to limit accesses to storage to generate result sets.
+</p>
+
+
+
+<p>
+  Configuration:
+</p>
+<table class="table">
+  <tr id="fast-search">
+    <th style="white-space:nowrap;">fast-search</th>
+    <td>
+        Use to speed up search -
+        see <a href="performance/feature-tuning.html#when-to-use-fast-search">when to use fast-search</a>.
+        An attribute is a linear array-like data structure -
+        matching documents hence means scanning <em>all</em> attribute values.
+        Setting <a href="reference/schema-reference.html#attribute">fast-search</a>
+        generates a B-tree index structure for quicker lookup,
+        using more memory, and more CPU when updating.
+        It increases steady state memory usage for all attribute types
+        and also add initialization overhead for numeric types.
+<pre>
+schema foo {
+    document bar {
+        field titles type array&lt;string&gt; {
+            indexing:  summary | attribute
+            attribute: fast-search
+        }
+    }
+}
+</pre>
+    </td>
+  </tr>
+  <tr id="fast-access">
+    <th style="white-space:nowrap;">fast-access</th>
+    <td>
+      Use to increase feed rates by having replicas on all nodes in memory.
```

Comment:
Consider being more explicit on when _fast-access_ makes sense using.

Reply:
good point! there is some more details in the linked-to doc